_summaries/liu2019large-scale_2.md

```diff
@@ -1,19 +1,90 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
-# Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+paper: liu2019large-scale
+author: Shivi Agarwal
+score: 9
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+
+## The Core Idea of this paper?
+Real world data is long tailed and open-ended distributed. A practical recognition system must classify among majority and minority classes. 
+This paper discusses about the Open Long-Tailed Recognition (OLTR) as learning from naturally distributed data and optimizing the classification 
+accuracy over a balanced test set including head, tail, and open classes. OLTR focuses on imbalanced classification, few-shot learning, and open-set 
```

Comment:
Might be good to define what head, tail, and open classes are for people who may be less familiar with the terminology. There is a nice figure (figure 1) in the paper that could be paired with a sentence or two to nicely describe these 3 types of classes.

Reply:
Added as per your comment.